Add SSH certificates documentation (#4592)
theforeman/foreman-documentation: guides/common/modules/proc_disabling-ssh-certificate-authentication-on-smart-proxy.adoc

:_mod-docs-content-type: PROCEDURE

[id="disabling-ssh-certificate-authentication-on-smart-proxy_{context}"]
= Disabling SSH certificate authentication on {SmartProxy}

> You can disable SSH certificate authentication on {SmartProxy} and revert to plain SSH key pair authentication.

Reviewer: ```suggestion
If you disable SSH certificate authentication on {SmartProxy}, you will revert to plain SSH key pair authentication.
```
